clang/include/clang/Basic/Attr.td

```diff
@@ -1944,6 +1944,13 @@ def ReturnsTwice : InheritableAttr {
   let SimpleHandler = 1;
 }
 
+def UnnamedAddr : InheritableAttr {
+  let Spellings = [Clang<"unnamed_addr">];
+  let Subjects = SubjectList<[Function]>;
+  let Documentation = [Undocumented];
```

Comment:
I think for new attributes, we generally do prefer to document them. This attribute especially seems like it would benefit from documentation, so please add some for it. Doesn’t have to be much, just a sentence or two explaining that it adds the LLVM attribute and an example would be enough imo.

Reply:
i added documentation